medusa/providers/nzb/anizb.py

```diff
@@ -118,5 +121,33 @@ def _get_size(self, item):
         """Override the default _get_size to prevent it from extracting using the default tags."""
         return try_int(item.get('size'))
 
+    def _get_episode_search_strings(self, episode, add_string=''):
+        """Get episode search strings."""
+        if not episode:
+            return []
+
+        search_string = {
+            'Episode': []
+        }
+
+        for show_name in allPossibleShowNames(episode.show, season=episode.scene_season):
+            episode_string = show_name + '*'
+
+            # If the showname is a season scene exception, we want to use the indexer episode number.
+            if show_name in get_scene_exceptions(episode.show.indexerid, season=episode.scene_season,
+                                                 indexer=episode.show.indexer):
+                # This is apparently a season exception, let's use the scene_episode instead of absolute
+                ep = int(episode.scene_episode)
+            else:
+                ep = int(episode.scene_absolute_number)
+            episode_string += str(ep)
+
+            if add_string:
+                episode_string += ' ' + add_string
```

Comment:
Isn't the `add_string` for proper tags? This is an anime provider, this isn't a need for this right?